src/core/util/threadpool.cpp

```diff
@@ -31,6 +31,10 @@
 #include <inviwo/core/util/raiiutils.h>
 #include <inviwo/core/util/stdextensions.h>
 
+#ifdef WIN32
+#include <windows.h>
```

Comment:
Should we have those #define nominmax and lean and what ever it is here? or do we set that globally?

Reply:
They are defined at https://github.com/inviwo/inviwo/blob/8503e60c6a176ae50627ce979bcf5fc688bf86af/cmake/compileoptions.cmake#L121-L122